### ISSUE TITLE
Toggle stopped working

### ISSUE THREAD

**Issue by Skowryk** [erikras/redux-form-material-ui#124]
Toggle stopped working, all time on off position with no option to turn it on

**Comment by nmaves** [comment]
You need to give/leave more information on this.  We need to see code examples and even better a working one on jsbin etc.

**Comment by Skowryk** [comment]
Well I've been using it and everything was ok until new release, tested it with just regular redux form field
<Field
                name='vatDeductable'
                labelPosition='right'
                component={Toggle}
                label={<FormattedMessage id='Invoices.Form.Label.deductable'/>}
              />
and simple redux form
SettlementForm = reduxForm(
  {
    form: 'invoiceGeneral'
  }
)(SettlementForm);

It's always off now and cannot be turned on across the entire website

**Comment by nmaves** [comment]
Could it be related to #102 ?

**Comment by tom3012** [comment]
Same problem in my project! With v4.1.2 the Toggle works as expected. With v4.1.3 Toggle stays at disabled position. No special code, just regular field as in comment of @Skowryk

**Comment by baldman** [comment]
I am experiencing the same issue. In my case, the toggle takes value from initialValues first (so no defaultToggled), however, does not seem to have any bearing. The toggle does not work at all right now.

**Comment by werg27** [comment]
Same problem here, upgrade to 4.1.3 and toogle doesnt work :s
-> yarn add redux-form-material-ui@4.1.2 and its work ;)

**Comment by damonmaria** [comment]
Same story here. Every single Toggle on the site. Fixed in disabled position. Clicking does nothing. Downgrading to 4.1.2 fixes it.
```js
import { Field } from 'redux-form'
import { TextField, SelectField, Toggle } from 'redux-form-material-ui'

      <Field
        component={Toggle}
        name="inOperation"
        label={vocab.InOperation}
        labelPosition="right"
      />
```

**Comment by jmenglis** [comment]
I saw the most recent update and made a comment on it.  I didn't have time but wanted to look into the proper fix for it.

**Comment by liviuignat** [comment]
@erikras I had to crate my own toggle to make it work. It does not work with form `initialValues`, because you need to set `defaultToggled={value}` instead of `toggled={value}`.

Code snippet:

```js
const FormToggle = ({
  input: {value, ...inputProps },
  onChange = () => {},
  ...props
}: any) => (
  <Toggle
    {...mapError(props)}
    {...inputProps}
    defaultToggled={value}
    onChange={(event, newValue) => {
      inputProps.onChange(newValue);
      onChange(newValue);
    }}
  />
);

export default (props) => (
  <Field
    {...props}
    component={FormToggle}
  />
);
```

**Comment by nmaves** [comment]
Check out my PR above #126, I believe that should fix things.

Can anyone test it out for me?

**Comment by erikras** [comment]
Fix published in [`v4.1.4`](https://github.com/erikras/redux-form-material-ui/releases/tag/v4.1.4).

**Comment by damonmaria** [comment]
For me, with v4.1.4, the Toggle now correctly responds to touch, and I can check it and uncheck it (this did not work in v4.1.3).

But, it does not start checked if the value passed in `initialValues` is `true`. If I switch back to v4.1.2 they do start in the correct state.
